decidim-generators/lib/decidim/generators.rb

```diff
@@ -6,7 +6,7 @@ module Decidim
   module Generators
     def self.edge_git_branch
       if Decidim::Generators.version.match?(/\.dev$/)
-        "develop"
+        "feature/new-collaborative_texts-module" # back to "develop" once https://github.com/decidim/decidim/pull/13978 is accepted
```

Comment:
I will apply this suggestion. Is created for visibility. 
```suggestion
        "develop"
```